src/metabase/models/card.clj

```diff
@@ -570,6 +571,10 @@
   [_card]
   [:name (serdes/hydrated-hash :collection) :created_at])
 
+(defmethod mi/exclude-internal-content-hsql :model/Card
+  [_model & {:keys [table-alias]}]
+  [:not= (h2x/identifier :field (some-> table-alias name) :creator_id) config/internal-mb-user-id])
```

Comment:
```suggestion
  [:not= (h2x/identifier :field table-alias :creator_id) config/internal-mb-user-id])
```
`h2x/identifier` will turn it to if it's non-nil so you don't need to do it.

Reply:
same with all other places

Reply:
The point of this is to call `name` so the function can take a keyword `table-alias`, which feels more consistent with HoneySQL than requiring `table-alias` to be a string.

Reply:
`h2x/identifier` allows components to be both string or keyword; it'll convert everything to string in the end

Reply:
I just tested this, you're right. I could have sworn there was a test failing on the keyword somewhere though. I will try the test suite again